app/models/container_quota.rb

```diff
@@ -2,5 +2,6 @@ class ContainerQuota < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   belongs_to :container_project
 
+  has_many :container_quota_scopes, :dependent => :destroy
```

Comment:
@cben you mention in the BZ `keep history instead of overwriting in refresh.` [ref](https://bugzilla.redhat.com/show_bug.cgi?id=1504560#c12) is that something that you want to accomplish in this PR?  Should this disconnect instead of destroy the quota scopes?

What is a quota scope BTW?

Reply:
see answer below https://github.com/ManageIQ/manageiq/pull/16655#issuecomment-352514133